boskos/client/client.go

```diff
@@ -31,6 +32,7 @@ type Client struct {
 	url       string
 	resources []string
 	owner     string
+	lock      sync.Mutex
```

Comment:
lock should be above the fields it guards. If it guards all fields it should be embedded. if it guards a non-obvious subset, subset should be documented with a comment.